app/move/controllers/create/personal-details.test.js

```diff
@@ -135,6 +135,41 @@ describe('Move controllers', function() {
       })
     })
 
+    describe('#savePerson()', function() {
+      const mockId = '1234567'
+
+      context('without id', function() {
+        beforeEach(function() {
+          sinon.spy(personService, 'create')
```

Comment:
You could move these spies to a shared `beforeEach` to reduce some repetition.

I also wonder if they may be safer as stubs so that the actual function doesn't still get called.